RecoTracker/LST/python/lstSeedTracks_cfi.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+initialStepSeedTracks = cms.EDProducer(
```

Comment:
these definitions should be moved to some common place where either trackingNtuple or LST can pick up.
Plain snapshot copy seems like a bad idea.
In particular, reusing the existing producer names is dangerous, as it would conflict with  `trackingNtuple_cff`

Alternatively, can we still import from `trackingNtuple_cff` (without customise function calls) ?

Reply:
These are not plain copies of the ones used in ` trackingNtuple_cff`. The naming has been changed(simplified), so there can be no conflicts.

But I get your point, these could be shared, even with the same name, if they are defined somewhere else. Is `trackingNtuple_cff` a good place for that? These are created on the fly by the customization function but I could try a way so that this is not needed.

Reply:
> These are not plain copies of the ones used in ` trackingNtuple_cff`. The naming has been changed(simplified), so there can be no conflicts.

ah, OK then. Perhaps add `lst` (prefix/affix) to be more clear that these are not generic.
At some point we should skip this seed->track conversion step. It's mostly unnecessary.